app/src/main/java/org/mozilla/fenix/quickactionsheet/QuickActionSheetStore.kt

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.quickactionsheet
+
+import mozilla.components.lib.state.Action
+import mozilla.components.lib.state.State
+import mozilla.components.lib.state.Store
+
+typealias QuickActionSheetStore = Store<QuickActionSheetState, QuickActionSheetAction>
+
+data class QuickActionSheetState(
```

Comment:
Let's add KDocs :) similar to https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/search/SearchStore.kt